third_party/terraform/resources/resource_container_registry.go

```diff
@@ -0,0 +1,109 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceContainerRegistry() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceContainerRegistryCreate,
+		Read:   resourceContainerRegistryRead,
+		Delete: resourceContainerRegistryDelete,
+
+		Schema: map[string]*schema.Schema{
+			"location": {
+				Type:     schema.TypeString,
+				Default:  "US",
```

Comment:
```suggestion
```

Can we omit this so that the "global" location is the default?

Reply:
Yeah, the default location is also `US` but without a prefix
`gcr.io hosts images in data centers in the United States, but the location may change in the future`

I'll change it